### PR TITLE
ENH: network connectivity up/down detection and corresponding toasts

### LINKED ISSUE #521: Detect if the internet connection went down/up and show corresponding toasts

### PR DESCRIPTION
**Pull request by deepansh96**
Fixes #521 

## Test Plan

- [ ] Test Responsiveness
   - [ ] Laptop (1200px)
   - [ ] Tablet (760px)
   - [ ] Phone (320px)
- [ ] Cross-Browser Testing
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Safari (use BrowserStack)
- [x] Local Language Support
- [ ] Wrote tests
- [x] Tested locally
- [x] Self-review
- [x] Comments have been added appropriately
- [x] Check for bundle size [here](https://bundlephobia.com/) if adding a package
- [x] Added relevant details like Labels/Projects/Milestones etc.
- [ ] Images have `alt` attributes
- [ ] Any `target="_blank"` links have `rel="noopener"`
- [ ] Only SVGs are used as images. If PNGs are used, their size has been optimised.
- [ ] Any SVGs without text have their `aria-label` attributes set
- [ ] Tested on staging
- [ ] Tested on an actual physical phone
- [ ] Tested on production

### CODE REVIEW

File: src/assets/locales/en.js [hunk 1]
```diff
@@ -357,7 +357,8 @@ export default {
     },
     create_plio: "Could not create Plio. Please try again",
     generic: "Something went wrong. Please try again!",
-    internet: "Please check your internet connection",
+    internet_lost: "Your device lost it's internet connection",
+    internet_found: "Your device is connected to the internet!",
```
Comment (dalmia) [inline src/assets/locales/en.js:361]: would it be simpler to say:

internet connection lost, internet connection restored?

Comment (deepansh96) [reply src/assets/locales/en.js:361]: Haan makes sense, will change

File: src/assets/locales/en.js [hunk 2]
```diff
@@ -357,7 +357,8 @@ export default {
     },
     create_plio: "Could not create Plio. Please try again",
     generic: "Something went wrong. Please try again!",
-    internet: "Please check your internet connection",
+    internet_lost: "Your device lost it's internet connection",
```
Comment (deepansh96) [inline src/assets/locales/en.js:360]:
```suggestion
    internet_lost: "Internet connection lost",
```

File: src/assets/locales/en.js (hunk 1 above)
Comment (deepansh96) [reply src/assets/locales/en.js:361]:
```suggestion
    internet_lost: "Internet connection lost",
    internet_found: "Internet connection restored",
```

File: src/assets/locales/hi.js [hunk 3]
```diff
@@ -358,7 +358,8 @@ export default {
     },
     create_plio: "प्लायो बनाते हुए कुछ गड़बड़ हो गयी",
     generic: "कुछ दिक्कत आ गयी कृपया दोबारा कोशिश करें",
-    internet: "कृपया अपने इंटरनेट कनेक्शन की जाँच करें",
+    internet_lost: "आपके डिवाइस ने अपना इंटरनेट कनेक्शन खो दिया है",
+    internet_found: "आपका डिवाइस इंटरनेट से जुड़ा है!",
```
Comment (deepansh96) [inline src/assets/locales/hi.js:362]:
```suggestion
    internet_lost: "इंटरनेट कनेक्शन टूट गया है",
    internet_found: "इंटरनेट कनेक्शन वापस आ गया है",
```

**Pull request merged**